css/main.css

```
@@ -3,69 +3,69 @@
     padding: 0;
 }
 
-html{
+html {
     scroll-behavior: smooth;
 }
 
-html, body{
+html, 
+body {
    
-    width:100%;
+    width: 100%;
     font-family: 'Roboto', sans-serif;
     font-weight: 300;
     
 }
 
-.centrar{
+.centrar {
     margin-left: auto;
```

I think this would have the same effect on the element.

```suggestion
    margin: 0  auto;
```

---

done.